components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetMacros.xml

```diff
@@ -257,6 +257,5 @@
     )))
   #end
 #end
-{{/velocity}}
-</content>
+{{/velocity}}</content>
```

Comment:
I would have put white space changes in a different commit, since they're not  directly related to the issue.